docs/management/connectors/action-types/servicenow-sir.asciidoc

```diff
@@ -1,25 +1,83 @@
 [role="xpack"]
 [[servicenow-sir-action-type]]
-=== ServiceNow connector and action
+=== ServiceNow SecOps connector and action
 ++++
 <titleabbrev>ServiceNow SecOps</titleabbrev>
 ++++
 
-The ServiceNow SecOps connector uses the https://docs.servicenow.com/bundle/orlando-application-development/page/integrate/inbound-rest/concept/c_TableAPI.html[V2 Table API] to create ServiceNow security incidents.
+The {sn} SecOps connector uses the https://docs.servicenow.com/bundle/orlando-application-development/page/integrate/inbound-rest/concept/c_TableAPI.html[V2 Table API] to create {sn} security incidents.
+
+[float]
+[[servicenow-sir-connector-prerequisites]]
+==== Prerequisites
+After upgrading from {stack} version \<=7.15.0 to version >=7.16.0, you must complete the following within your {sn} instance before creating a new {sn} SecOps connector or <<servicenow-sir-connector-update, updating an existing one>>:
```

Comment:
I find this notation a little hard to parse:  <=7.15.0 to version >=7.16.0

Reply:
Would something like the following read better:
After upgrading from Elastic Stack version 7.15.0, or an earlier one, version 7.16.0 or later, you must...

@jmikell821 do you have a preference for one or the other?

Reply:
@nastasha-solomon alt suggestion: 

`After upgrading from Elastic Stack version 7.15.0 or earlier to version 7.16.0 or later`

Reply:
I'll use that for this and the Security docs then. Thanks, @gchaps and @jmikell821!